cli/internal/pkcs11/pkcs11_integration_test.go

```diff
@@ -0,0 +1,172 @@
+//go:build integration && pkcs11
+
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: BUSL-1.1
+*/
+
+package pkcs11
+
+import (
+	"crypto"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"flag"
+	"testing"
+	"time"
+
+	"github.com/ThalesGroup/crypto11"
+	"github.com/edgelesssys/marblerun/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var configPath = flag.String("pkcs11-config", "", "path to PKCS#11 configuration file")
+
+func TestLoadX509KeyPair(t *testing.T) {
+	// Ensure we can load the PKCS#11 configuration file
+	pkcs11, err := crypto11.ConfigureFromFile(*configPath)
+	require.NoError(t, err)
+	require.NoError(t, pkcs11.Close())
+
+	testCases := map[string]struct {
+		init    func(t *testing.T) (keyID, keyLabel, certID, certLabel string)
+		wantErr bool
+	}{
+		"identified by ID and label": {
+			init: func(t *testing.T) (keyID, keyLabel, certID, certLabel string) {
+				t.Helper()
+				require := require.New(t)
+				prefix := t.TempDir()
```

Comment:
The idea was to easily generate a unique ID/label for each subtest.
Replaced it with `uuid.New()` since we don't really need anything but a random string